MSOE.MediaComplete.Lib/Library/DataSource/FileSystem.cs

```diff
@@ -73,6 +74,7 @@ private FileSystem()
                 NotifyFilter = NotifyFilters.LastWrite | NotifyFilters.FileName | NotifyFilters.DirectoryName,
                 IncludeSubdirectories = true
             };
+            _watcher.InternalBufferSize = 32768;
```

Comment:
Can we abstract 32768 to a constant variable and give a comment explaining why it's 32768 for people other than Tyler and I